secure-releases.md

```diff
@@ -0,0 +1,60 @@
+# Secure Releases and CVE Management for the JS Ecosystem
+
+## Secure Releases
+
+In order for your JS software project to be safe, secure, and reliable, the versions it releases need to follow security best practices. Below are the OpenJS Foundations recommended best practices for secure releases.
+
+### Two-factor Authentication
+
+Two-factor authentication (2FA) means having something more than just your password to confirm you're you. The second factor can be a one-time code using an app like Google Authenticator, a hardware key (such as Yubikey or Titan), or a passkey using touch ID, face ID, or something else that's stored locally on your device. Note: do not *ever* use SMS for 2FA unless it's the only mechanism offered, since SMS is vulnerable to SIM swap and number forgery attacks.
+
+Although this extra authentication step can be a mild inconvenience, the benefit for security is overwhelmingly worth it. Both Github and npm will soon require 2FA for every user, so you might as well enable it now.
+
+### Preparing for Publish
+
+It can be easy to forget to run steps before publishing. If you have any build steps (transpiling with Babel or TypeScript, emitting types with TypeScript, etc) and/or validation steps (running tests, etc), make sure they're set up in the `scripts` section of `package.json` such that `npm run prepublish` or `npm run prepublishOnly` would invoke them - that way, running `npm publish` in the future will automatically run these steps.
+
+Publishing to npm can be done in one step (`npm publish`), or two (`npm pack` to generate a tarball, and then later, `npm publish path/to/tarball` to publish). Even if you publish separately with one step, it's a good idea to run `npm pack` in CI to ensure that the steps succeed, and that the tarball is somewhat reproducible. (TODO: do this in my packages' CI and link to it here as an example)
+
+### Publishing
+If you have only 1 person who can publish releases, then this step is optional. If you have 2 or more, this step is strongly recommended.
+
+Publish a packed tarball from CI, only upon an explicit request. This means, don't make a new release automatically, but make sure that a human has intentionally said "it's time to publish a new version, here's the version number it will use". This explicit request can be a pushed semver-versioned git tag, or an explicit user-run workflow - but does not include merging/committing to the default branch.
```

Comment:
```suggestion
Publish a packed tarball from CI, only upon an explicit request. This means, don't make a new release automatically, but make sure that a human has intentionally said "it's time to publish a new version, here's the version number it will use". This explicit request can be a pushed SemVer-conformant git tag, or an explicit user-run workflow - but does not include merging/committing to the default branch.
```

Reply:
I think this correction would actually be incorrect, since a leading `v` isn't conformant with SemVer, but it's the most common idiom in a semver-versioned git tag. Additionally, in a monorepo, git tags are often something like `subpackage@1.2.3`, which is also not Semver-conformant.

Reply:
I'm confused by this response. All I meant was that the short form of  Semantic Versioning is SemVer, not Semver and not semver.

Reply:
oh perhaps we are getting wires crossed because I'm suggesting two things:
semver -> SemVer
and
versioned -> conformant

Reply:
right. re capitalizing semver, i'm not actually referring to the spec, I'm referring to the `semver` npm module, which is the actual source of truth for the JS ecosystem. re "conformant", for the reasons indicated, versioned tags are almost never conformant with the spec.

Reply:
ok.  I don't have strong feelings here